apps/crn-server/test/data-providers/contentful/research-output.data-provider.test.ts

```diff
@@ -354,6 +354,48 @@ describe('Research Outputs Data Provider', () => {
       expect(result!.documentType).toEqual('Grant Document');
     });
 
+    test('Should default missing working group reference to an empty array of teams', async () => {
```

Comment:
didnt you mean 
```suggestion
    test('Should default an empty teams collection to an empty array of teams', async () => {
```
or something like that?